packages/components/src/components/tabs/tabs.tsx

```diff
@@ -52,10 +76,11 @@ const Tabs = ({
         }
     }, []);
 
-    let initial_index_to_show, tab_width;
+    let initial_index_to_show = 0;
+    let tab_width;
```

Comment:
```suggestion
    let tab_width: string;
```
@jim-deriv Not sure tho 🙇🏻